.npmignore

```diff
@@ -1,5 +1,4 @@
 node_modules/
-src/
 .git*
 .idea
 .project
```

Comment:
It is recommended to use `files` in `package.json` instead of `.npmignore` https://medium.com/@jdxcode/for-the-love-of-god-dont-use-npmignore-f93c08909d8d

Reply:
This is generated by npm publish in old days using `.gitignore` as template. We can remove that.